app/imports/collections/applicationSections.js

```diff
@@ -0,0 +1,13 @@
+import { Meteor } from 'meteor/meteor';
+
+class ApplicationSectionsCollection extends Mongo.Collection {
+  insert(doc, callback) {
+    doc.createdAt = doc.createdAt || new Date();
+    doc.updatedAt = doc.updatedAt || new Date();
+    doc.userId = Meteor.userId();
+    doc.year = doc.year || new Date().getFullYear();
```

Comment:
its fine to leave. however only the application needs it right, not the sections?